packages/create-sitecore-jss/src/templates/nextjs/scripts/generate-plugins.ts

```diff
@@ -14,21 +12,12 @@ import { getItems } from './utils';
   This can be customized in writePlugins().
 */
 
-enum ModuleType {
-  CJS,
-  ESM,
-}
-
 interface PluginDefinition {
```

Comment:
Why not take it one step further: move this interface as well, and make `writePlugins` take an array of `PluginDefinition`s?